src/system-health/health_checker/config.py

```diff
@@ -0,0 +1,105 @@
+import os
+import json
+from sonic_device_util import get_machine_info
+from sonic_device_util import get_platform_info

```

Comment:
sonic_device_util is being deprecated in favor of the new sonic-py_common package. Please update accordingly. See https://github.com/Azure/sonic-buildimage/pull/5043 for reference.

Reply:
Sure